### ISSUE TITLE
RuntimeError in DevicesController#show

### ISSUE THREAD

**Issue by FedeG** [lelylan/devices#68]
#### Information
When i go to: [`http://api-proxy.lelylan.dev.exoip.com.ar/devices/5898c7fc6899264eba000001?access_token=084539f2401f6e7ced4a0ffc822c5ba7e4cb3ea4ba5f249bbb7d0be99a6e23f0&token_type=bearer&expires_in=86400&state=remember-me`](`http://api-proxy.lelylan.dev.exoip.com.ar/devices/5898c7fc6899264eba000001?access_token=084539f2401f6e7ced4a0ffc822c5ba7e4cb3ea4ba5f249bbb7d0be99a6e23f0&token_type=bearer&expires_in=86400&state=remember-me`)
The api respond: **Called id for nil, which would mistakenly be 4 -- if you really wanted the id of nil, use object_id**.

#### Response page:
![captura de pantalla de 2017-02-06 16-58-57](https://cloud.githubusercontent.com/assets/4097554/22664759/d685f4ee-ec90-11e6-83a1-7467be704c4a.png)

#### Error detail:
This error is generated from : `app/controllers/devices_controller.rb:68:in 'find_owned_resources'`

##### In **app/controllers/devices_controller.rb**:
```ruby
66: def find_owned_resources
67:   @devices = Device.where(resource_owner_id: current_user.id)
68: end
```

##### In **app/controllers/application_controller.rb**:
```ruby
7: def current_user
8:   @current_user ||= User.find(@device.resource_owner_id)          if request.headers['X-Physical-Secret']
9:   @current_user ||= User.find(doorkeeper_token.resource_owner_id) if request.headers['Authorization']
10:   return @current_user
11: end
```

###### Variables used in **current_user**:
```
@device.resource_owner_id -> nil
request.headers['X-Physical-Secret'] -> nil
doorkeeper_token.resource_owner_id -> 5894d5c59f6fb8bb72000001d
request.headers['Authorization'] -> nil
```

The variable **doorkeeper_token.resource_owner_id** has correct user id **5894d5c59f6fb8bb72000001d** but **request.headers['Authorization']** is nil, then, **@current_user** not set in **current_user** function

**Comment by FedeG** [comment]
https://github.com/lelylan/devices/issues/67

**Comment by andreareginato** [comment]
I think this can be considered closed.

**Comment by FedeG** [comment]
Yes, this bug are fixed in #66
